Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.h

```diff
@@ -26,6 +27,7 @@ namespace AzToolsFramework
 
             /// @cond
             ComponentModeCollection() = default;
+            ComponentModeCollection(ViewportEditorModeTrackerInterface* viewportEditorModeTracker);
```

Comment:
nit: Please add `explicit` for single argument constructor

Reply:
fixed 👍